bundle/Resources/config/ezplatform.yml

```diff
@@ -0,0 +1,6 @@
+system:
+    default:
+        field_templates:
+            - { template: "@NetgenEzPlatformSiteApi/content_fields.html.twig", priority: 10000 }
```

Comment:
Ideally, these two priorities in this file should be set to `0`, so developers overriding the field rendering do not need to look up what priority to use to correctly override the field block.

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/94/commits/9ef3816b914ccd9e113cac622859da340f68b89c